### DISCUSSION TITLE
Where can talk to people about implementations - I have questions about modules but not sure GitHub is best for this.

### DISCUSSION THREAD

**Discussion by kairos0ne** [Moddable-OpenSource/moddable#1131] [General]
Do you have a discord/ slack or other channel where we can discuss implementing moddable and its various modules ?

**Comment by phoddie** [comment]
Welcome!

I've converted your issue to a GitHub Discussion, which can work well for talking about implementations. We also have a [Gitter](https://app.gitter.im/#/room/#embedded-javascript_moddable:gitter.im) which is useful for real-time conversations. I personally prefer GitHub Discussions because they are a little more organized and much easier to search.

**Comment by kairos0ne** [comment]
Thanks I will open a discussion then :) and close this one.
